packages/server/migrations/20240321154545_create_employees_table.js

```diff
@@ -4,10 +4,10 @@
  */
 exports.up = function (knex) {
   return knex.schema.createTable('Employees', (table) => {
-    table.increments('id').primary();
+    table.increments('id').primary().unique();
```

Comment:
Same here, if you will create new migration files to update the schema, you will not need these changes.